java/sage/VideoFrame.java

```diff
@@ -5639,20 +5640,64 @@ private boolean performLanguageMatch(String defaultLang, String[] availLangs, in
 
   private int getMatchingLangIndex(String[] langOptions, String targetLang)
   {
+    
+    int selectedIndex = -1;  
+      
     targetLang = targetLang.toLowerCase();
     for (int i = 0; i < langOptions.length; i++)
     {
       if (langOptions[i].toLowerCase().indexOf(targetLang) != -1)
-        return i;
+      {
+        selectedIndex = i;
+        
+        //Prefer subtitle tracks that are not forced
+        if(langOptions[i].toLowerCase().indexOf("[" + Sage.rez("forced") + "]") != -1)
```

Comment:
This is preferring forced subtitle tracks...the inverse of the comment.

Reply:
Opps.  😔